cinema/serializers.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField()
+    description = serializers.CharField()
```

Comment:
Code Style issue: You have specified a `CharField` without a `max_length`. If you are going to specify a `max_length`, then it's more reasonable to use `CharField` instead of `TextField`. If you don't need to specify a `max_length`, then you should use `TextField`. Please revise your design and make the necessary changes.